### ISSUE TITLE
images sampled were gray

### ISSUE THREAD

**Issue by Koid217** [DeweiHu/OCT_DDPM#8]
Hi, I used your method to process other types of medical images, but the images sampled were gray. Do you know what happened

**Comment by Koid217** [comment]
Hello, I used it to process my own data set and found that its loss function has been floating in the hundreds or thousands, do you know why this is

**Comment by DeweiHu** [comment]
Hi, may I ask what modality you are working on? I would recommend try to normalize the intensity range to [1,3]. In the ckpts folder I uploaded a model I trained which is compatible with this intensity range. Maybe you can use it as a pre-trained model for transfer learning.

**Comment by Koid217** [comment]
Hello, I am processing the CT image, but the effect is not very good, the whole image has become slightly gray, I think it may be caused by the relatively high quality of the initial image

**Comment by Koid217** [comment]
In fact, I switched to [1,3], but it still didn't converge

**Comment by gguyinging** [comment]
我也出现了这样的问题，我用的是图片进行测试，我的损失值只收敛到了一百左右，这会是为什么呢？
